keep/providers/axiom_provider/axiom_provider.py

```diff
@@ -38,11 +38,10 @@ class AxiomProviderAuthConfig:
 class AxiomProvider(BaseProvider):
     """Enrich alerts with data from Axiom."""
 
+    webhook_documentation_here_differs_from_general_documentation = True
```

Comment:
can you elaborate on this? why not just making it align? i'm confused

Reply:
Some documentation in the docs is much better and more detailed than documentation in the provider. Pulling massive documentation back to the provider is a separate task, so decided to keep it this way